nixos/modules/services/misc/calibre-server.nix

```diff
@@ -13,20 +13,34 @@ in
   ###### interface
 
   options = {
-
     services.calibre-server = {
-
       enable = mkEnableOption "calibre-server";
 
       libraryDir = mkOption {
         description = ''
           The directory where the Calibre library to serve is.
-          '';
-          type = types.path;
+        '';
+        type = types.path;
       };
 
-    };
+      listenAddress = mkOption {
+        default = "::";
+        example = "127.0.0.1";
+        description = ''
+          The interface on which to listen for connections. The value "::" will
```

Comment:
Should we change it to `The address on which to listen for connections`

Reply:
Could do - I was keeping it inline with the upstream documentation for now, but don't feel strongly either way.

Reply:
OK. Thanks for the explanation.